aQute.libg/src/aQute/lib/home/Home.java

```diff
@@ -0,0 +1,21 @@
+package aQute.lib.home;
+
```

Comment:
This class seems overkill and no cohesive (Bnd things, maven things...)

Reply:
This seems to be more a bndlib than a aQute.lib thing.

Reply:
It is used by `aQute.libg/src/aQute/lib/settings/Settings.java` which is not in bndlib.